### PR TITLE
strip blockchain - backend

### PR DESCRIPTION
**Pull request by pldespaigne**

### CODE REVIEW

File: apps/backend-functions/src/main.ts [hunk 1]
```diff
@@ -150,20 +150,20 @@ export const generateDeliveryPDF = functions.https.onRequest(onGenerateDeliveryP
 //--------------------------------
 //            RELAYER           //
 //--------------------------------
+// ! STRIP BLOCKCHAIN CODE
```
Comment (lsenta) [inline apps/backend-functions/src/main.ts]: keep

File: apps/backend-functions/src/main.ts [hunk 2]
```diff
@@ -150,20 +150,20 @@ export const generateDeliveryPDF = functions.https.onRequest(onGenerateDeliveryP
...
+// };
 
-const RELAYER_CONFIG: RelayerConfig = {
-  ...relayer,
-  mnemonic
-};
-
-export const relayerDeploy = functions.runWith({timeoutSeconds: 540}).https
-  .onCall((data, context) => relayerDeployLogic(data, RELAYER_CONFIG));
+// export const relayerDeploy = functions.runWith({timeoutSeconds: 540}).https
```
Comment (lsenta) [inline apps/backend-functions/src/main.ts]: keep

File: apps/backend-functions/src/main.ts [hunk 3]
```diff
@@ -150,20 +150,20 @@ export const generateDeliveryPDF = functions.https.onRequest(onGenerateDeliveryP
...
+//   .onCall((data, context) => relayerDeployLogic(data, RELAYER_CONFIG));
 
-export const relayerRegister = functions.runWith({timeoutSeconds: 540}).https
-  .onCall((data, context) => relayerRegisterENSLogic(data, RELAYER_CONFIG));
+// export const relayerRegister = functions.runWith({timeoutSeconds: 540}).https
```
Comment (lsenta) [inline apps/backend-functions/src/main.ts]: keep, these are entry point triggered in the front, just disable the call in the front.

File: apps/backend-functions/src/orgs.ts [hunk 4]
```diff
@@ -59,20 +59,22 @@ export async function onOrganizationUpdate(
     const { userIds } = before as OrganizationDocument;
     const admin = await db.collection('users').doc(userIds[0]).get().then(adminSnapShot => adminSnapShot.data()!); // TODO use laurent's code after the merge of PR #698
 
-    const orgENS = emailToEnsDomain(before.name.replace(' ', '-'), RELAYER_CONFIG.baseEnsDomain);
+    if (false) { // ! STRIP BLOCKCHAIN CODE
```
Comment (lsenta) [inline apps/backend-functions/src/orgs.ts]: 👍

File: apps/backend-functions/src/main.ts [hunk 5]
```diff
@@ -150,20 +150,20 @@ export const generateDeliveryPDF = functions.https.onRequest(onGenerateDeliveryP
 //--------------------------------
 //            RELAYER           //
 //--------------------------------
+// ! STRIP BLOCKCHAIN CODE
+// const RELAYER_CONFIG: RelayerConfig = {
+//   ...relayer,
+//   mnemonic
+// };
```
[the remark this answers is not in the document]
Comment (pldespaigne) [reply apps/backend-functions/src/main.ts]: I was missing a packages and just `npm i`, should I delete the package-lock from the PR ?

File: apps/backend-functions/src/main.ts [hunk 6]
```diff
@@ -28,7 +29,6 @@ import {
   onDocumentWrite,
   onOrganizationDocumentUpdate
 } from './utils';
-import { mnemonic, relayer } from './environments/environment';
```
Comment (lsenta) [inline apps/backend-functions/src/main.ts]: shitty change ;)

File: apps/backend-functions/src/main.ts [hunk 7]
```diff
@@ -150,7 +150,6 @@ export const generateDeliveryPDF = functions.https.onRequest(onGenerateDeliveryP
 //--------------------------------
 //            RELAYER           //
 //--------------------------------
-
```
Comment (lsenta) [inline apps/backend-functions/src/main.ts]: same here

**Pull request merged**